vimeo/VimeoNetworking
# AF-316: Updates SwiftLint to inlcude the "opening_brace" rule

### Ticket
[AF-316](https://vimean.atlassian.net/browse/AF-316)

### Pull Request Checklist

### Ticket Summary
Update SwiftLint rules by adding the `opening_brace` rule. 

### Implementation Summary
- Added `opening_brace` to our `.swiftlint.yml` file.
- Built the project, allow the autocomplete option to update all opening braces in the project.

### How to Test
The changes here should be strictly related to style. Unit tests should continue to run and pass. If you feel any opening brace was moved incorrectly, or if a particular case was missed, let me know.

It's also worth noting that because this change touches so many files that it could make viewing Git blame more difficult. We won't lose any history here, but this commit will essentially appear on every file in the project.
